app/src/main/java/co/nimblehq/ui/screens/second/SecondFragment.kt

```diff
@@ -0,0 +1,56 @@
+package co.nimblehq.ui.screens.second
+
+import android.content.Intent
+import android.provider.MediaStore
+import co.nimblehq.R
+import co.nimblehq.ui.base.BaseFragment
+import co.nimblehq.ui.screens.home.Data
+import kotlinx.android.synthetic.main.fragment_second.*
+
+/**
+ * TODO update with a new permission requesting
+ */
+//@RuntimePermissions
+class SecondFragment : BaseFragment<SecondViewModel>() {
+
+    override val viewModelClass = SecondViewModel::class
+
+    override val layoutRes: Int = R.layout.fragment_second
+
+    override fun setupView() {
+        btOpenCamera.setOnClickListener { openCamera() }
+    }
+
+    override fun bindViewModel() {
+        viewModel.setPersistedData() bindTo ::bindPersistedData
+    }
+
+    //    @NeedsPermission(Manifest.permission.CAMERA)
+    fun openCamera() {
+        startActivity(Intent(MediaStore.ACTION_IMAGE_CAPTURE))
+    }
+//
+//    @OnPermissionDenied(Manifest.permission.CAMERA)
+//    fun showDeniedForCamera() {
+//        Toast.makeText(this, "Permission camera denied", Toast.LENGTH_SHORT).show()
```

Comment:
copy/paste and comment still left in here? 🤔

Reply:
I think we could update to use `RxPermission` instead 🤔

Reply:
why in this PR? 🤔

Reply:
@sleepylee screen migration step from activity to fragment, will be handled in next PR 👌